mono/mini/debugger-agent.h

```diff
@@ -28,6 +28,7 @@ struct _MonoDebuggerCallbacks {
 	void (*user_break) (void);
 	void (*debug_log) (int level, MonoStringHandle category, MonoStringHandle message);
 	gboolean (*debug_log_is_enabled) (void);
+	void (*send_crash) (char *jsonDump, MonoStackHash *hashes, int pauseTime);
 };
```

Comment:
The parameter names should follow the runtime coding conventions.

Reply:
It's now snakecase